README.md

```diff
@@ -8,114 +8,201 @@
+
+* Enter a username, tick the "Provide user access to the AWS Management Console - optional" checkbox, and click "I want to create an IAM user". This will enable programmatic access through access keys.
+
+   <img src=https://raw.githubusercontent.com/ballerina-platform/module-ballerinax-aws.sns/main/docs/resources/create-user-iam-user.png alt="Create IAM user" width="50%">
+
+* Click through the permission setup, and add the user to the user group we previously created.
+
+   <img src=https://raw.githubusercontent.com/ballerina-platform/module-ballerinax-aws.sns/main/docs/resources/create-user-set-permission.png alt="Attach user group" width="50%">
+
+* Review the details and click "Create user."
+
+   <img src=https://raw.githubusercontent.com/ballerina-platform/module-ballerinax-aws.sns/main/docs/resources/create-user-review.png alt="Review user" width="50%">
+
+3. Generate access key ID and secret access key
+
+* Once the user is created, you will see a success message. Navigate to the "Users" tab, and select the user you created.
+
+   <img src=https://raw.githubusercontent.com/ballerina-platform/module-ballerinax-aws.sns/main/docs/resources/view-user.png alt="View User" width="50%">
+
+* Click on the "Create access key" button to generate the access key ID and secret access key.
+
+   <img src=https://raw.githubusercontent.com/ballerina-platform/module-ballerinax-aws.sns/main/docs/resources/create-access-key.png alt="Create access key" width="50%">
+
+* Follow the steps and download the CSV file containing the credentials.
+
+   <img src=https://raw.githubusercontent.com/ballerina-platform/module-ballerinax-aws.sns/main/docs/resources/download-access-key.png alt="Download credentials" width="50%">
 
-**Note**: Ensure you follow the [prerequisites](https://github.com/ballerina-platform/module-ballerinax-aws.sns#set-up-aws-sns-api) to set up the AWS SNS API.
+## Quickstart
 
-To use the `aws.sns` connector in your Ballerina application, modify the `.bal` file as follows:
+To use the `aws.sns` connector in your Ballerina project, modify the `.bal` file as follows:
 
 ### Step 1: Import the connector
+
 Import the `ballerinax/aws.sns` package into your Ballerina project.
 ```ballerina
 import ballerinax/aws.sns;
 ```
 
 ### Step 2: Instantiate a new connector
-Create a `sns:ConnectionConfig` record with the obtained `accessKeyId` and `secretAccessKey` and initialize the connector with it.
-```ballerina
-sns:ConnectionConfig config = {
-    accessKeyId: accessKeyId,
-    secretAccessKey: secretAccessKey,
-    region: region
-};
 
-sns:Client amazonSNSClient = check new(config);
+Instantiate a new `sns` client using the access key ID, secret access key, and region.
+```ballerina
+sns:Client sns = check new({
+    credentials: {
+        accessKeyId: "ACCESS_KEY_ID",
+        secretAccessKey: "SECRET_ACCESS_KEY"
+    },
+    region: "REGION"
+});
```

Comment:
Here rather than hard coding values shall we define set of configurables and use them. You can follow this pattern [1] 

[1] - [https://github.com/ballerina-platform/module-ballerinax-docusign.dsesign?tab=readme-ov-file#step-2-instantiate-a-new-connector](https://github.com/ballerina-platform/module-ballerinax-docusign.dsesign?tab=readme-ov-file#step-2-instantiate-a-new-connector)